src/bionics.cpp

```diff
@@ -459,7 +463,11 @@ void player::power_bionics()
                                         power_only_desc.str() );
             }
             ypos += fold_and_print( w_description, ypos, 0, DESCRIPTION_WIDTH, c_ltblue,
-                                    bionics[( *current_bionic_list )[cursor]->id].description );
+                                    bionics[( *current_bionic_list )[cursor]->id].description ) + 1;
+            ypos += fold_and_print( w_description, ypos, 0, DESCRIPTION_WIDTH, c_ltgray,
+                                    list_occupied_bps( ( *current_bionic_list )[cursor]->id,
+                                    _( "This bionic occupies the following body parts:" ),
+                                    ( getmaxy( w_description ) > ypos + ( int )num_bp + 1 ) ).c_str() );
```

Comment:
It's quite hard to comprehend the meaning without looking at the declaration of `list_occupied_bps()`
A boolean flag can make the task much easier:
```c++
const bool one_per_line = ypos + ( int )num_bp + 1 < getmaxy( w_description ); 
...
list_occupied_bps( ..., one_per_line ).c_str() 
```